Add -Werror to all Build Jobs

```
/home/markus/Projekte/Elektra/current/src/libs/elektra/keyset.c: In function ‘elektraOpmphmInvalidate’:
/home/markus/Projekte/Elektra/current/src/libs/elektra/keyset.c:48:47: warning: unused parameter ‘ks’ [-Wunused-parameter]
 static void elektraOpmphmInvalidate (KeySet * ks)
                                               ^~
/home/markus/Projekte/Elektra/current/src/libs/elektra/keyset.c: In function ‘elektraOpmphmCopy’:
/home/markus/Projekte/Elektra/current/src/libs/elektra/keyset.c:65:41: warning: unused parameter ‘dest’ [-Wunused-parameter]
 static void elektraOpmphmCopy (KeySet * dest, const KeySet * source)
                                         ^~~~
/home/markus/Projekte/Elektra/current/src/libs/elektra/keyset.c:65:62: warning: unused parameter ‘source’ [-Wunused-parameter]
 static void elektraOpmphmCopy (KeySet * dest, const KeySet * source)
                                                              ^~~~~~
```

--- a/src/libs/elektra/keyset.c
+++ b/src/libs/elektra/keyset.c
@@ -45,7 +45,7 @@
  *
  * @param ks the KeySet
  */
-static void elektraOpmphmInvalidate (KeySet * ks)
+static void elektraOpmphmInvalidate (KeySet * ks ELEKTRA_UNUSED)
 {
 #ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
 	if (ks && ks->opmphm) opmphmClear (ks->opmphm);
@@ -62,7 +62,7 @@ static void elektraOpmphmInvalidate (KeySet * ks)
  * @param dest the destination KeySet
  * @param source the source KeySet
  */
-static void elektraOpmphmCopy (KeySet * dest, const KeySet * source)
+static void elektraOpmphmCopy (KeySet * dest ELEKTRA_UNUSED, const KeySet * source ELEKTRA_UNUSED)
 {
 #ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
 	if (!source || !dest)

